### PR TITLE
Feature/add better daemon info

### PR DESCRIPTION
**Pull request by clatko-globe**

### CODE REVIEW

File: microcosm_pubsub/tests/test_dispatcher.py [hunk 1]
```diff
@@ -47,6 +48,35 @@ def test_handle_message_succeeded(self):
...
+            )
+            assert_that(
+                result,
+                has_properties(
+                    elapsed_time=greater_than(0.0),
+                    result=MessageHandlingResultType.SUCCEEDED,
+                ),
+            )
+            mocked_logger.log.assert_called_with(ANY, ANY, exc_info=None, extra={
+                'media_type': 'application/vnd.microcosm.derived',
```
Comment (GabeLambda) [inline microcosm_pubsub/tests/test_dispatcher.py]: Double-quotes.

File: microcosm_pubsub/tests/test_dispatcher.py [hunk 2]
```diff
@@ -47,6 +48,35 @@ def test_handle_message_succeeded(self):
...
+                'X-Request-Ttl': '665',
+                'message_id': 'message-id',
+                'handler': 'Function',
+                'X-Request-Handler': 'function',
+                'X-Request-Daemon': 'Unknown',
```
Comment (GabeLambda) [inline microcosm_pubsub/tests/test_dispatcher.py]: Maybe add one more test where you override the existing METRICS_NAME?

**Comment by jameschung2019** [comment]
Hey there, please close this PR or merge it appropriately. It will be auto-closed in 48 hours if not acted upon.

If you decide to merge it, comment here and take ownership to coordinate, schedule, and get it done. Thanks!

**Comment by clatko-globe** [comment]
this PR is getting us closer to jaeger, so it would be obsolete once jaeger or similar is added. probably best not to delete for now.

**Pull request closed without merging**